runtime/cudaq/qis/execution_manager.h

```diff
@@ -194,17 +194,36 @@ class ExecutionManager {
 
 /// Get the default execution manager instance.
 ExecutionManager *getExecutionManager();
```

Comment:
Q: Should we replace `getExecutionManager` with the new `get_execution_manager`?

Reply:
Why do we need a new `get_execution_manager() : ExecutionManager&` when we could just keep using the original one?

Reply:
Discussed offline... The original method is defined by the expansion of the macro below.